docs/animations/withTiming.mdx

```diff
@@ -53,6 +53,7 @@ The value on which the animation will come at rest.
 The timing animation configuration.
 
 import { useTimingPlayground } from "@site/src/components/ModifierPlayground";
+import { useInterpolateColorPlayground } from "@site/src/components/ModifierPlayground";
```

Comment:
You can add a new `interpolateColor.mdx` page to Advanced APIs section. If you don't want it to be shown after merge, but the code to be on main, you can prefix it with `_` like so: `_interpolateColor.mdx`.

That way you can iterate on the feature and merge it to main without it being seen by the end user.

Reply:
Sure, should I document it, or you will take the lead?

Reply:
Okay, let's maybe merge this PR and create a new branch where we could document this function